modules/swagger-codegen/src/main/resources/python/api.mustache

```diff
@@ -1,31 +1,8 @@
 # coding: utf-8
 
-"""
-{{classname}}.py
-Copyright 2015 SmartBear Software
```

Comment:
@webron @fehguy any concern from Smartbear perspective to remove the software license from the auto-generated Python code?